etc/config/analysis.defaults.properties

```diff
@@ -0,0 +1,16 @@
+compilers=&llvm-mca
+compilerType=analysis
+supportsBinary=false
+objdumper=
+# TODO: We might want to set a demangler, to deal with pasted compiler output with mangled symbols
+demangler=
+
+group.llvm-mca.compilers=llvm-mcatrunk
+group.llvm-mca.groupName=LLVM Machine Code Analyzer
+group.llvm-mca.supportsBinary=false
+
+compiler.llvm-mcatrunk.name=llvm-mca (trunk)
+#compiler.llvm-mcatrunk.exe=/usr/bin/llvm-mca
+compiler.llvm-mcatrunk.exe=/Users/filcab/dev/llvm/build/bin/llvm-mca
```

Comment:
Ok, so few things here.
You probably want to delete this line 14, and uncomment line 13, so the "default" path is used.
Then you want to add the `.amazon.` file back with the `/opt/...` path instead, so we can separate "3rd party users" from we running in the servers :)